Update and fix instructions to build the docs (#1583)
simpeg/simpeg: docs/content/getting_started/contributing/documentation.rst

Building the documentation
~~~~~~~~~~~~~~~~~~~~~~~~~~

> You can build the documentation pages locally to check how new changes would
> look like. First, make sure that you have :ref:`created and activated an

Reviewer: ```suggestion
You can build the documentation pages locally to see how the new changes will look. 
First, make sure that you have :ref:`created and activated an
```

Reviewer: The original statement just sounded odd to me...

Also your `:ref:` link is broken
